src/cudafeatbin/compute-fbank-feats-cuda.cc

```diff
@@ -106,6 +106,11 @@ int main(int argc, char *argv[]) {
       num_utts++;
       std::string utt = reader.Key();
       const WaveData &wave_data = reader.Value();
+      if (wave_data.SampFreq() != fbank_opts.frame_opts.samp_freq) {
+        KALDI_ERR << "File: " << utt << " has an incompatible sampling "
```

Comment:
To me, "incompatible" implies that there exists a sample rate which is "compatible" with that specified in the config file. While, strictly speaking it's true (there is exactly one), this usually suggests that there is more than than one. How about "incorrect", "unexpected", "bad" or  "mismatching"?

×4 in total.